apps/stitch/prisma/schema.prisma

```diff
@@ -87,3 +87,16 @@ model Plan {
   createdAt      DateTime @default(now())
   updatedAt      DateTime @updatedAt
 }
+
+model Customer {
```

Comment:
saving locally, it can avoid hitting the payment provider and it will work if the payment provider is out